### PR TITLE
fix: disable integration tests

### PR DESCRIPTION
**Pull request by popas90**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

Disable integration tests that are currently failing. Fixing them is tracked in another work item.

### Notes for the reviewer

N/A

### Screenshots

N/A

### CODE REVIEW

File: .circleci/config/@config.yml [hunk 1]
```diff
@@ -53,49 +53,49 @@ workflows:
...
+      #     <<: *staging_branch_only_filter
+      # - eks_integration_tests:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - aks_integration_tests:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - integration_tests_operator_on_k8s:
```
Comment (iuliams) [inline .circleci/config/@config.yml:76]: I <think> this can be fully removed. @kat1906 can you confirm? :-/

**Review by iuliams — APPROVED** [review]

File: .circleci/config/@config.yml (hunk 1 above)
Comment (minsiyang) [reply .circleci/config/@config.yml:76]: Let's disable it now now, and have the removing part in a different PR 🙏🏻

Comment (kat1906) [reply .circleci/config/@config.yml:76]: I haven't started working on that et so can stay for now :)

**Review by kat1906 — APPROVED** [review]

**Pull request closed without merging**